ui/js/directives/treeherder/bottom_nav_panel.js

```diff
@@ -34,67 +34,3 @@ treeherder.directive('thRelatedBugQueued', function () {
         templateUrl: 'partials/main/thRelatedBugQueued.html'
     };
 });
-
-treeherder.directive('thFailureClassification', [
-    'thClassificationTypes',
-    function (thClassificationTypes) {
-        return {
-            scope: {
-                failureId: "=",
-                jobResult: "="
-            },
-            link: function (scope) {
-                scope.$watch('[failureId, jobResult]', function () {
-                    if (scope.failureId) {
-                        scope.classification = thClassificationTypes.classifications[scope.failureId];
-                        scope.hoverText=scope.classification.name;
-                        scope.iconCls = (scope.failureId === 7 ? "fa-star-o" : "fa fa-star") +
-                                         " star-" + scope.jobResult;
-                    }
-                });
-            },
-            template: '<span title="{{hoverText}}">' +
-                '<i class="fa {{iconCls}}"></i>' +
-                '</span> {{hoverText}}'
-        };
-    }]);
-
-treeherder.directive('thSimilarJobs', [
-    'ThJobModel',
-    function (ThJobModel) {
-        return {
-            restrict: "E",
-            templateUrl: "partials/main/similar_jobs.html",
-            link: function (scope) {
-                scope.$watch('job', function (newVal) {
-                    if (newVal) {
-                        scope.update_similar_jobs(newVal);
-                    }
-                });
-                scope.similar_jobs = [];
-                scope.similar_jobs_filters = {
-                    machine_id: true,
-                    job_type_id: true,
-                    build_platform_id: true
-                };
-                scope.update_similar_jobs = function (job) {
-                    var options = { result_set_id__ne: job.result_set_id };
-                    angular.forEach(scope.similar_jobs_filters, function (elem, key) {
-                        if (elem) {
-                            options[key] = job[key];
-                        }
-                    });
-                    ThJobModel.get_list(scope.repoName, options).then(function (data) {
-                        scope.similar_jobs = data;
-                    });
-                };
-            }
-        };
-    }]);
-
-treeherder.directive('thPinboardPanel', function () {
```

Comment:
Unused as of https://github.com/mozilla/treeherder/commit/20e85c20#diff-785d18b1ec6858c43b406520d1d7d478L6